source/scss/_library/_objects.media.scss

```diff
@@ -25,13 +25,13 @@
 \*------------------------------------*/
 
 // 3:2 ratio
-.u-img-ratio--3x2 {
+[class*=u-img-ratio] {
   position: relative;
+  background-color: #f0eeea;
```

Comment:
Mind using one of the neutral color vars for this? `$c-gray--light` should do it.